driver-core/src/main/java/com/datastax/driver/core/QueryLogger.java

```diff
@@ -746,6 +760,61 @@ protected String parameterValueAsString(ColumnDefinitions.Definition definition,
         return valueStr;
     }
 
+    protected int appendParameters(SimpleStatement statement, StringBuilder buffer, int remaining) {
```

Comment:
There is quite a bit of duplication of code between the appendParameters/parameterValueAsString methods for the SimpleStatement and BoundStatement implementations.  However, they are probably different enough to justify it being the way it is.